rules/fixtures/rules_indeterminates_without_alertname.yaml

```diff
@@ -0,0 +1,18 @@
+groups:
+  - name: indeterminate
+    rules:
+      - record: job:http_requests:rate1m
+        expr: sum by (job)(rate(http_requests_total[1m]))
+      - record: job:http_requests:rate5m
+        expr: sum by (job)(rate(http_requests_total[5m]))
+      - record: job:http_requests:rate15m
+        expr: sum by (job)(rate(http_requests_total[15m]))
+      - record: job:http_requests:rate30m
+        expr: sum by (job)(rate(http_requests_total[30m]))
+      - record: job:http_requests:rate1h
+        expr: sum by (job)(rate(http_requests_total[1h]))
+      - record: job:http_requests:rate2h
+        expr: sum by (job)(rate(http_requests_total[2h]))
+      # This shouldn't run in parallel because of the missing alertname label
+      - record: alerts
+        expr: count(ALERTS)
```

Comment:
It could be parallelized as well where we determine the parallel order of everything _except_ ALERTS and then ALERTS run at the end

Reply:
In this case there's no alertname label matcher for `ALERTS`. To keep it simple I initially decided to consider it an indeterminate, but it's true that we can also optimise this case and just make it dependent on every other alerting rule. Done it here: https://github.com/prometheus/prometheus/pull/17064/commits/e9d4be3b1f007cbde72de1c8b9ca75ed2331c09a